mappings/net/minecraft/recipe/RecipeMatcher.mapping

```diff
@@ -0,0 +1,121 @@
+CLASS net/minecraft/class_1662 net/minecraft/recipe/RecipeMatcher
+	COMMENT Matching class that matches a recipe to it's required resources.
+	COMMENT This specifically does not check patterns (See {@link ShapedRecipe} for that.)
+	FIELD field_7550 inputs Lit/unimi/dsi/fastutil/ints/Int2IntMap;
+	METHOD method_20478 addInput (Lnet/minecraft/class_1799;I)V
+		COMMENT Adds an item stack to the pool of available resources.
+		ARG 1 stack
+			COMMENT The item stack to submit.
+		ARG 2 maxCount
+	METHOD method_7400 addInput (Lnet/minecraft/class_1799;)V
+		COMMENT Adds a full item stack to the pool of available resources.
+		COMMENT
+		COMMENT This is equivalent to calling {@code addInput(stack, ItemStack.MAX_COUNT)}
```

Comment:
```suggestion
		COMMENT <p>This is equivalent to calling {@code addInput(stack, ItemStack.MAX_COUNT)}</p>
```

Reply:
That effect does adding the paragraph tags achieve? Seems unnecessary.

Reply:
It adds a paragraph break to the generated javadoc. Otherwise these will appear in the same paragraph.